src/app/users/users.service.ts

```diff
@@ -73,33 +75,42 @@ export class UsersService {
         switchMap(userIdentity => this.httpClient.put<UserPermission[]>(
           // TODO: Update permissions for given scope via specific API call, e.g.
           // PUT /users/{userId}/permissions/{scope}
```

Comment:
I am in doubt whether we should remove the TODO here. In fact `UserPermissionRequest` contains the `scope` field of the type `UserPermissionScope`. Since we want the `PUT` endpoint to let the client application update more than one type of permission therefore there has to be a way to pass more than one scope. We cannot achieve such a goal with a path parameter. Consequently we have to pass the scopes in the body of a request.

In my opinion we have to change the TODO to be `TODO: Update permissions for given scope via specific API call, e.g. PUT /users/{userId}/outbound-permissions with scopes to update in a request body`. And this is what is implemented at the moment.
@svetivanova what do you think? Should we remove the TODO from here?

Reply:
I totally agree with you, so i removed TODO